config.yaml

```diff
@@ -7,6 +7,11 @@ options:
     description: "Amount of time, in ticks, to allow followers to connect and sync to a leader."
     type: int
     default: 5
+  snap-channel:
+    type: string
+    default: "rock/edge"
+    description: |
+      "Kafka snap channel to install and track."
```

Comment:
I would add to the description, mentioning that this cannot be changed during runtime (based on other review comments).